### PR TITLE
Fix dev admin/profile + some comments of PR dev to main

### PR DESCRIPTION
**Pull request by Alice-Berthelot**

### CODE REVIEW

File: client/src/pages/Profile.jsx [hunk 1]
```diff
@@ -30,19 +30,24 @@ function Profile() {
...
-        <section className="profile-delete-button-wrapper">
-          <button
-            type="submit"
-            aria-label="Supprimer le profil"
-            className="profile-delete-button"
-            onClick={handleOpenModal}
-          >
-            Supprimer mon profil
-          </button>
+        <section className="profile-desktop-wrapper">
```
Comment (Dinosaure7) [inline client/src/pages/Profile.jsx:33]: Do we keep <section> and <article> here?

**Review by sorrento974 — APPROVED** [review]

File: client/src/styles/Profile.css [hunk 2]
```diff
@@ -112,6 +112,12 @@
   margin-bottom: 0.8rem;
 }
 
+.profile-delete-button-wrapper {
+  display: flex;
+  justify-content: flex-end;
+  margin-right: 0rem;
```
Comment (Dinosaure7) [inline client/src/styles/Profile.css:118]: Why margin-right at 0?

File: client/src/styles/Profile.css [hunk 3]
```diff
@@ -123,12 +129,6 @@
...
 }
 
-.profile-delete-button-wrapper {
-  display: flex;
-  justify-content: flex-end;
-  margin-right: 1rem;
-}
-
 .profile-information-edit-article > input:focus-within {
   border: 12px solid var(--third-color);
```
Comment (Dinosaure7) [inline client/src/styles/Profile.css:133]: We like rem better than px

File: client/src/styles/Profile.css [hunk 4]
```diff
@@ -291,21 +291,31 @@
     margin-top: 5rem;
     border: none;
     border-radius: 0;
```
Comment (Dinosaure7) [inline client/src/styles/Profile.css:293]: Why border-radius at 0?

**Review by Dinosaure7 — CHANGES_REQUESTED** [review]
Some small changes

File: client/src/pages/Profile.jsx (hunk 1 above)
Comment (Alice-Berthelot) [reply client/src/pages/Profile.jsx:33]: yes we said this morning that we keep this, and we will see later to change everything with display-grid

File: client/src/styles/Profile.css (hunk 2 above)
Comment (Alice-Berthelot) [reply client/src/styles/Profile.css:118]: to remove the margin that has been set for the same element in mobile version

File: client/src/styles/Profile.css (hunk 3 above)
Comment (Alice-Berthelot) [reply client/src/styles/Profile.css:133]: until now we have used px for the borders

File: client/src/styles/Profile.css (hunk 4 above)
Comment (Alice-Berthelot) [reply client/src/styles/Profile.css:293]: to remove the border-radius that has been set for the same element in mobile version

File: client/src/pages/Login.jsx [hunk 5]
```diff
@@ -41,13 +41,11 @@ function Login() {
...
 
-      <article>
-        <img
-          src="../src/assets/images/mockup.png"
-          alt="Mockups de l'application"
-          className="imgMockup"
-        />
-      </article>
+      <img
+        src="../src/assets/images/mockup.png"
```
Comment (Delgorithm) [inline client/src/pages/Login.jsx]: You can add the image in a "const" and then use it here

**Review by Delgorithm — APPROVED** [review]

File: client/src/pages/Login.jsx (hunk 5 above)
Comment (Alice-Berthelot) [reply client/src/pages/Login.jsx]: done

File: client/src/components/ArtDetails.jsx [hunk 6]
```diff
@@ -15,21 +15,22 @@ function ArtDetails({ art, onClose }) {
...
+          alt={art.title}
+          className="modal-picture-artDetails"
+        />
+
         <h2>
-          {art.title ? art.title.toUpperCase() : "TITRE - "}
+          {art.title && art.title.toUpperCase()}
           {art.title && art.artist && " - "}
-          {art.artist ? art.artist : "Artiste"}
+          {art.artist && art.artist}
```
Comment (Yoann-Fortin) [inline client/src/components/ArtDetails.jsx]: Is not needed

File: client/src/components/ArtDetails.jsx [hunk 7]
```diff
@@ -15,21 +15,22 @@ function ArtDetails({ art, onClose }) {
...
         </h2>
-        <p>
-          {art.description
-            ? art.description
-            : "Aucune description n'est disponible pour le moment."}
+        <p className={art.description && "modal-artDetails-description"}>
+          {art.description && art.description}
```
Comment (Yoann-Fortin) [inline client/src/components/ArtDetails.jsx]: Is not needed

File: package.json [hunk 8]
```diff
@@ -32,7 +35,11 @@
     "concurrently": "^8.2.2",
     "cross-env": "^7.0.3",
     "husky": "^9.0.11",
-    "validate-branch-name": "^1.3.0",
-    "lint-staged": "^15.2.2"
+    "lint-staged": "^15.2.2",
+    "validate-branch-name": "^1.3.0"
+  },
+  "dependencies": {
```
Comment (Yoann-Fortin) [inline package.json]: These dependencies should be in the client's .json package. It's amazing that it even passed the linter

**Review by Yoann-Fortin — CHANGES_REQUESTED** [review]

File: client/src/components/ArtDetails.jsx (hunk 6 above)
Comment (Alice-Berthelot) [reply client/src/components/ArtDetails.jsx]: done

File: client/src/components/ArtDetails.jsx (hunk 7 above)
Comment (Alice-Berthelot) [reply client/src/components/ArtDetails.jsx]: done

File: package.json (hunk 8 above)
Comment (Alice-Berthelot) [reply package.json]: we have the same problem on most of our PR, can we check this together?

**Review by Yoann-Fortin — APPROVED** [review]

**Pull request merged**